pypeerassets/kutil.py

```diff
@@ -43,15 +43,11 @@ def load_network_parameters(self, query: str):
 
     def wif_to_privkey(self, wif: str) -> bytes:
         '''import WIF'''
+        if 51 < len(wif) < 52:
+            return 'Invalid WIF length'
 
         b58_wif = b58decode(wif)
-
-        if len(wif) == 51:
-            return {'privkey': hexlify(b58_wif[1:-4]).decode(), 'net_prefix': hexlify(b58_wif[0:1])}
-        if len(wif) == 52:
-            return {'privkey': hexlify(b58_wif[1:-5]).decode(), 'net_prefix': hexlify(b58_wif[0:1])}
-        else:
-            return 'Invalid WIF length'
+        return {'privkey': b58_wif, 'net_prefix': hexlify(b58_wif[0:1])}
```

Comment:
Getting rid of the hexlify and decode makes sense for privkey return. When defining the privkey to return does this take into consideration the required difference when wif length is 51 to use b58_wif[1:-4] and when it is 52 to use b58_wif[1:-5] ?